src/server/components/dashboard/controllers.ts

```diff
@@ -349,3 +372,151 @@ export async function feedbackController(
     next(error);
   }
 }
+
+export async function listsEditAnnualReviewDateController(
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<void> {
+  try {
+    const { listId } = req.params;
+    const list = await findListById(listId);
+    const annualReviewStartDate = formatAnnualReviewDate((list as List), "annualReviewStartDate");
+    const maxDate = list?.jsonData.annualReviewStartDate ? getMaxDate(list?.jsonData.annualReviewStartDate): "";
+    const formattedMaxDate = maxDate ? format(maxDate, DATE_FORMAT) : "";
```

Comment:
this date needs to be calculated in exactly the same way as used when validating in the  `listsEditAnnualReviewDatePostController`.  Currently they're out of sync.